ambuda/models/user_permissions.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime
+from enum import IntEnum
+from sqlalchemy import (
+    Column,
+    DateTime,
+    Integer,
+)
+
+from ambuda.models.base import Base, foreign_key, pk
+
+
+class UserPermissionValues(IntEnum):
```

Comment:
Hmm, there are a few approaches to representing a small set of ~enumerated values in a db:

1. ENUM column -- generally inflexible if adding more values later
2. Lookup table -- create a new table and refer to it with foreign keys. This also lets us add metadata
3. ENUM in code, as is done here

I don't know what's best between (2) and (3), but we currenty do (2) for page statuses, and if there's not a strong reason for (3) I prefer consistency. Or in other words, something like 1+2+4 here:

https://zoharpeled.wordpress.com/2020/11/07/role-based-security-database-design/

Reply:
(PS: to be clear, I don't know the right answer to this and am happy to go with this PR's implementation if there's some prior art for it.)

Reply:
I agree with this choice for (1) consistency and for (2) easy expansion with additional roles. I pushed a new commit to reflect this.